src/Jit/Parser.php

```diff
@@ -51,6 +51,21 @@ class Parser
      */
     protected $_states = [];
 
+    /**
+     * @var int
+     */
+    private $_T_ARROW_FUNCTION;
+
+    /**
+     * @var int
+     */
+    private $_T_DOUBLE_ARROW;
+
+    /**
+     * @var int
+     */
+    private $_T_ATTRIBUTE;
```

Comment:
new property need to have public modifier, unfortunatelly, ref https://3v4l.org/HYEML , mark use `@internal` to avoid abuse